tests/test_context.py

```diff
@@ -242,9 +242,12 @@ def test_one_context_call_with_simple_component(self):
             "{% load component_tags %}{% component_dependencies %}"
             "{% component name='incrementer' %}{% endcomponent %}"
         )
-        rendered = template.render(Context()).strip()
-
-        self.assertEqual(rendered, '<p class="incrementer">value=1;calls=1</p>', rendered)
+        rendered = template.render(Context()).strip().replace("\n", "")
+        self.assertEqual(
+            rendered,
+            '<link href="relative_file/relative_file.css" media="all" rel="stylesheet"><script src="relative_file/relative_file.js"></script><p class="incrementer">value=1;calls=1</p>',
+            rendered,
+        )
```

Comment:
This above also needs attention. For some reason, when I've added the `relative_file`, then I needed to update this piece of code. Do you know why?

Reply:
Not sure why. I don't see how the previous version of that test could ever have worked? 🤣 You version looks right, why didn't the component_dependencies render anything before?